rotkehlchen/chain/ethereum/modules/l2/zksync.py

```diff
@@ -0,0 +1,589 @@
+                self._query_and_save_transactions_for_range(
+                    address=address,
+                    start_ts=start_ts,
+                    end_ts=min_timestamp,
+                    from_hash=min_tx_hash,
+                    direction='older',
+                )
+                self._query_and_save_transactions_for_range(
+                    address=address,
+                    start_ts=max_timestamp,
+                    end_ts=end_ts,
+                    from_hash=max_tx_hash,
+                    direction='newer',
+                )
+            else:  # get the new transactions
+                self._query_and_save_transactions_for_range(
+                    address=address,
+                    start_ts=max_timestamp,
+                    end_ts=end_ts,
+                    from_hash=max_tx_hash,
+                    direction='newer',
+                )
+        except RemoteError as e:
+            log.error(
+                f'Got error "{e!s}" while querying zksync lite transactions '
+                f'from zksync api. Transactions not added to the DB '
+                f'address: {address} '
+                f'from_ts: {start_ts} '
+                f'to_ts: {end_ts} ',
+            )
+
+        return self._get_zksynctxs_db(
+            address=address,
+            start_ts=start_ts,
+            end_ts=end_ts,
+        )
+
+    def get_balances(
+            self,
+            addresses: Sequence[ChecksumEvmAddress],
+    ) -> dict[ChecksumEvmAddress, dict[Asset, Balance]]:
+        """May raise:
+           - RemoteError if there is a problem contacting the API
+        """
+        balances: defaultdict[ChecksumEvmAddress, dict[Asset, Balance]] = defaultdict(dict)
+        for address in addresses:
+            result = self._query_api(url=f'accounts/{address}')
+            try:
+                for symbol, raw_amount_str in result['finalized']['balances'].items():
+                    if (asset := self._get_token_by_symbol(symbol)) is None:
+                        log.error(f'Could not find asset for symbol {symbol} in zksync mapping')
+                        continue
+
+                    raw_amount = deserialize_int_from_str(
+                        symbol=raw_amount_str,
+                        location='zksync balances',
+                    )
+                    amount = asset_normalized_value(raw_amount, asset)
+                    try:
+                        usd_price = Inquirer().find_usd_price(asset)
+                    except RemoteError as e:
+                        log.error(
+                            f'Error processing zksync lite balance entry due to inability to '
+                            f'query USD price: {e!s}. Skipping balance entry',
+                        )
+                        continue
+
+                    balances[address][asset] = Balance(amount, usd_price)
+
+            except (KeyError, DeserializationError, RemoteError) as e:
```

Comment:
you already catch the remoterror in 564 right? Is there anything else raising it?

Reply:
It's due to `_get_token_by_symbol` also having the possibility of raising a remote error